crm/modules/user/user.inc.php

```diff
@@ -1377,7 +1377,7 @@ function user_page (&$page_data, $page_name, $options) {
 
 function json_web_token($uuid) {
     // Fetch JWT Secret
-    $jwtsecret = variable_get('jwtsecret');
+    $jwtsecret = variable_get('jwtsecret', $uuid);
```

Comment:
The function took a name as part of the signature, my first instinct was to try the $uuid as there were no other obvious choices and i got no errors. It might not be right, but the error went away.

Reply:
Whatever works best, i'm still learning PHP so i didn't know that was an option. I'm used to java and not allowing defaults in the signature. As long as you think it will work more effectively i'm game.